pkg/internal/resources/apiserver/apiserver.go

```diff
@@ -83,6 +84,7 @@ func Manifests(c Config) ([]unstructured.Unstructured, error) {
 								"--tls-cert-file=$(API_SERVER_TLS_CERT_FILE)",
 								"--tls-private-key-file=$(API_SERVER_TLS_PRIVATE_KEY_FILE)",
 								"--authentication-mode=$(AUTHENTICATION_MODE)",
+								"-v=$(VERBOSE)",
```

Comment:
```suggestion
								"-v=$(LOG_LEVEL)",
```

same to other places as well

Reply:
done